src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { Todo } from '../../Types/Todo';
+import usersFromServe from '../../api/users';
+import { getUserById } from '../../services/user';
```

Comment:
Avoid using `getUserById` directly in `TodoForm`. The user data should be prepared in the `App` component and passed as a prop if necessary.

Reply:
> Avoid using `getUserById` directly in `TodoForm`. The user data should be prepared in the `App` component and passed as a prop if necessary.

Good point, you can read about presentational and container components https://www.geeksforgeeks.org/reactjs-container-and-presentational-pattern-in-components/